dashboard/public/pixelation/pixelation.js

```diff
@@ -438,6 +445,18 @@ function setSliders() {
   drawGraph();
 }
 
+function setMinTextValues() {
+  if (heightText.value < 1) {
+    heightText.value = 1;
+  }
+  if (widthText.value < 1) {
+    widthText.value = 1;
+  }
+  if (bitsPerPixelText.value < 1) {
+    bitsPerPixelText.value = 1;
+  }
+}
```

Comment:
Yeah, I'm not sure of any ways to break it either. You could do something like:
```
heightRange.value = +heightText.value < 1 ? 1 : heightText.value
```
or if you were really concerned (which I don't think I am)
```
heightRange.value = parseInt(heightText.value, 10) < 1 ? 1 : heightText.value
```
Not sure either of those suggestions are really necessary though.  You could also make a little utility function i.e. ensureAtLeastOne that takes care of all this logic. Again, might be over-engineering. How you have it is probably fine :)